templates/BreadcrumbsTemplate.ss

```diff
@@ -1,6 +1,10 @@
-<% if $Pages %>
-    <nav class="breadcrumbs" aria-label="Breadcrumbs">
-        <ol class="breadcrumbs__list">
+<nav class="breadcrumbs" aria-label="Breadcrumbs">
+    <ol class="breadcrumbs__list">
+        <li class="breadcrumbs__list-item">
+            <a href="$BaseURL">Home</a>
```

Comment:
```suggestion
            <a href="$BaseURL"><%t StartupTheme.BREADCRUMBS_HOME 'Home' %></a>
```
I don't really mind what the actual localisation key is, I think this is the only hardcoded string in the whole theme so we don't have anything to match to.